libraryBuilds/com.fasterxml.jackson.core/com.fasterxml.jackson.core-jackson-core/build.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2015 the authors of j2objc-common-libs-e2e-test
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+    j2objcTranslation 'com.fasterxml.jackson.core:jackson-core:2.5.3:sources'
+}
+
+j2objcConfig {
+    // Almost always there are no tests provided in an external source jar.
+
+autoConfigureDeps true
+    forceFilenameCollisionCheck false
+    translateArgs = ['--segmented-headers', '--extract-unsequenced', '--build-closure']
```

Comment:
Please remove --build-closure from all of the files.  If it doesn't build without --build-closure, that means we aren't translating the source jar correctly and/or you don't have all the dependencies listed.